pkg/apis/extensions/install/install_test.go

```diff
@@ -78,10 +78,9 @@ func TestInterfacesFor(t *testing.T) {
 
 func TestRESTMapper(t *testing.T) {
 	gv := v1beta1.SchemeGroupVersion
-	hpaGVK := gv.WithKind("HorizontalPodAutoscaler")
 	daemonSetGVK := gv.WithKind("DaemonSet")
 
-	if gvk, err := registered.GroupOrDie(extensions.GroupName).RESTMapper.KindFor(gv.WithResource("horizontalpodautoscalers")); err != nil || gvk != hpaGVK {
+	if gvk, err := registered.GroupOrDie(extensions.GroupName).RESTMapper.KindFor(gv.WithResource("daemonsets")); err != nil || gvk != daemonSetGVK {
```

Comment:
Why this change?

This test should continue to pass without any change right?
Since extensions/v1beta1 is expected to continue to work